cosmwasm/packages/enclave-ffi-types/src/types.rs

```diff
@@ -16,50 +16,19 @@ pub struct EnclaveBuffer {
 }
 
 impl EnclaveBuffer {
+    pub fn null() -> Self {
```

Comment:
Might be helpful to use this to impl `::default()`?

Reply:
Yes, except it's shorter to write `EnclaveBuffer::null()` than `EnclaveBuffer::default()`, and it makes more sense in my head in this case. I can add it if you think it's more appropriate though.

Reply:
Lets make it ::default(). Just so it's consistent and slightly more clear what it's for